userspace/libsinsp/filter.cpp

```diff
@@ -2002,22 +2002,33 @@ sinsp_evttype_filter::~sinsp_evttype_filter()
 
 	m_catchall_evttype_filters.clear();
 
-	for(auto val : m_evttype_filters)
+	for(const auto &val : m_evttype_filters)
 	{
 		delete val.second->filter;
 		delete val.second;
 	}
 	m_evttype_filters.clear();
 }
 
+sinsp_evttype_filter::filter_wrapper::filter_wrapper()
+{
+	// Every rule is by default enabled for the default ruleset of 0.
+	enabled.resize(1);
```

Comment:
Let's be C++11-ish and do this with an initializer list

`sinsp_evttype_filter::filter_wrapper::filter_wrapper() : enabled{true} { ...`